dashboard/app/controllers/api/v1/pd/forms_controller.rb

```diff
@@ -4,12 +4,17 @@ def new_form
   end
 
   def create
-    form_data_hash = params.try(:[], :form_data) || {}
-    form_data_json = form_data_hash.to_unsafe_h.to_json.strip_utf8mb4
-
     form = new_form
+
+    form_data_hash = params.try(:[], :form_data)
+    form_data_json = form_data_hash ? form_data_hash.to_unsafe_h.to_json.strip_utf8mb4 : {}.to_json
     form.form_data_hash = JSON.parse(form_data_json)
 
+    status = params.try(:[], :status)
+    if status
```

Comment:
optional / for next time: I am not sure why the existing code is using params.try here, because params should always be defined in a controller method. `params[:status]` would be more clear.

Reply:
Ahh, good to know! I'll make a note of it and include in a future PR. Thanks.